scripts/lib/aws/sagemaker.sh

```diff
@@ -65,12 +66,12 @@ sagemaker_create_execution_role() {
 
   local __role_name="$1"
   daws iam create-role --role-name "$__role_name" --assume-role-policy-document '{"Version": "2012-10-17","Statement": [{ "Effect": "Allow", "Principal": {"Service": "sagemaker.amazonaws.com"}, "Action": "sts:AssumeRole"}]}' >/dev/null
-  assert_equal "0" "$?" "Expected success from aws iam create-role --role-name $__role_name but got $?" || exit 1
+  assert_equal "0" "$?" "Expected success from aws iam create-role --role-name $__role_name, got $?" || exit 1
```

Comment:
I mean, it's up to you, but any reason to change these assertion failure messages? Oftentimes I like to keep style-only changes like this in a separate PR that can reviewed apart from functional changes.